.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+
+      - run: npm ci
+
+      - run: npm run format:check
+
+      - name: Dry run publish
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: "quickbrownfox"
+          dryRun: true
```

Comment:
This is essentially the only change in this PR, the rest is cosmetic or was merged from #30